gorder.go

```diff
@@ -1,29 +1,24 @@
 package gorder
+import "errors"
```

Comment:
There's typically a double newline after a package declaration. Did you use `gofmt`/`goimports`?